release notes/v0.39.0.md

```diff
@@ -0,0 +1,32 @@
+k6 v0.39.0 is here! :tada: It's a small release that includes a bunch of bugfixes and minor enhancements. A lot of our focus was on some upcoming big changes, details for which you can read at the end of this document!
+
+## Enhancements and UX improvements
+
+- [#2274](https://github.com/grafana/k6/pull/2274) and [#2560](https://github.com/grafana/k6/pull/2560): The `csv` output supports a new `timeFormat` option (also configurable through the `K6_CSV_TIME_FORMAT` environment variable) with the possible values of `unix` (the default) and `rfc3399`. Thanks, @rpocklin!
+- [#2509](https://github.com/grafana/k6/pull/2509): Add `clear()` and `delete()` methods to the [`CookieJar` object](https://k6.io/docs/javascript-api/k6-http/cookiejar/) from the [`k6/http`](https://k6.io/docs/javascript-api/k6-http/) module. Thanks, @Maksimall89!
+- [#2282](https://github.com/grafana/k6/pull/2282): The iterations/s precision in the UI progress bars was increased. Thanks, @m3hm3t!
```

Comment:
we might want to mention all the other people who have worked on this

Reply:
Resolved by the latest [force push](https://github.com/grafana/k6/compare/9d682d82437985265e135cd6a24080be0b007dbd..6f763113a21481f8b3d72ac5496366cd5ffbd982)?